docs/_static/custom.css

```diff
@@ -22,3 +22,95 @@
   background-color: var(--pst-color-success);
   opacity: 0.1;
 }
+
+/* the badges to display colors to the end user */
```

Comment:
Let's put a link to the page that this is on so that it's clear what these rules are for